### ISSUE TITLE
After using MAKETHIN, TILT becomes 0 for elements who inherit non-zero TILT 

### ISSUE THREAD

**Issue by paa27** [MethodicalAcceleratorDesign/MAD-X#875]
How to reproduce the bug:

1. Define a parent class with non-zero TILT.
2. Define an element that inherits from the parent class.
3. Use MAKETHIN and run a TWISS: the slices will have TILT=0, regardless of parent's TILT.
	3.1 The edge elements will keep the correct TILT value for rbend

***REMARK***: This is not simply a bug in writing to the table. Elements actually behave like TILT=0.
	- e. g. (shown later) Given a sequence with an rbend with TILT=pi/2, after MAKETHIN D_y is 0 for the whole sequence and D_x gets extra contribution 

@ydutheil found that the bug seems to have been introduced in this commit:
~~~
41538b3c        madcern           Fri Mar 8 08:59:41 2019 +0100 Merge pull request #743 from tpersson/anotherThin
~~~

Here is a self contained example that reproduces the bug:

~~~
! Define element classes for a simple cell:
!--------------------------------------------!
b: rbend, l = 10, angle = 0.01;
btilt: rbend, l = 10, angle=0.01, tilt=pi/2;
qtilt: quadrupole, l = 1, k1 = -0.02, tilt=pi/2;
q: quadrupole, l = 1, k1 = 0.02;

! define the cell as a sequence:
!--------------------------------!

bbad: btilt; !This element will have the mentioned buggy behavior 
qbad: qtilt; !This element will also have the mentioned buggy behavior

sequ: sequence, l = 90;
b1: b, at = 20;
q1: qbad, at = 40;
b2: bbad, at = 60;
q2: q, at = 80;

ENDSEQUENCE;

! Thick twiss 
!-------------!

BEAM ,particle=proton;

USE, sequence=sequ;

SELECT, flag=twiss, column=name,keyword,tilt;
TWISS, sequence=sequ, betx=5,bety=3,file='thick.tfs';

PLOT, haxis=s, hmin=0, hmax=90, vaxis=dx,dy, table=twiss, file='disp';

!Thin twiss
!-----------!


SELECT, flag = makethin, clear;
SELECT, flag = makethin, class=rbend, slice=2;
SELECT, flag = makethin, class=quadrupole, slice=1;
USE, sequence=sequ;
MAKETHIN, sequence=sequ;

USE, sequence=sequ;
TWISS, sequence=sequ, betx=5,bety=3, file='thin.tfs';

PLOT, haxis=s, hmin=0, hmax=90, vaxis=dx,dy, table=twiss;


EXIT;
~~~

**Comment by tpersson** [comment]
I will have a look tomorrow. This must have been been introduced with the new makethin.

________________________________

**Comment by tpersson** [comment]
Fixed in #876
